docs/pages/database-access/guides/aws-dynamodb.mdx

```diff
@@ -0,0 +1,275 @@
+
+### EC2 instance role
+
+Visit the [IAM > Roles page](https://console.aws.amazon.com/iamv2/home#/roles) of
+the AWS Console, then press "Create Role". Under **Trusted entity type** select
+"AWS service". Under **Use case** select "EC2", then click **Next**.
+
+![Create Role to Identify EC2 Instance](../../../img/database-access/guides/dynamodb-create-ec2-role.png)
+
+On the "Add Permissions" page, you can simply click **Next** since this role does not require any permissions. In this guide, we will use the example name `TeleportDatabaseService` for this role. Once you have chosen a name, click **Create Role** to complete the process.
+
+### DynamoDB access role
+
+Navigate back to the Roles page and create a new role. Select the "AWS account"
+option, which creates a default trust policy to allow other entities in this
+account to assume this role:
+
+![Create Role Step 1](../../../img/database-access/guides/dynamodb-create-role-1.png)
+
+Click **Next**. Find the AWS-managed policy `AmazonDynamoDBFullAccess` and then select the policy:
+
+![Create Role Step 2](../../../img/database-access/guides/dynamodb-create-role-2.png)
+
+<Admonition type="note" title="Apply least-privilege permissions">
+The `AmazonDynamoDBFullAccess` policy may grant more permissions than desired.
+If you want to use a different IAM policy to reduce permissions, refer to
+[Managing access permissions to your Amazon DynamoDB 
+Resources](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/access-control-overview.html)
+for more information.
+</Admonition>
+
+Click **Next**. On the next page, enter a role name. In this guide we'll use
+the example name `ExampleTeleportDynamoDBRole` for this role.
+
+Under "Select trusted entities", update the JSON to allow the `TeleportDatabaseService`
+role to assume this role:
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": [
+                    "arn:aws:iam::(=aws.aws_access_key=):role/TeleportDatabaseService"
+                ]
+            },
+            "Action": "sts:AssumeRole",
+            "Condition": {}
+        }
+    ]
+}
+```
+
+Finally, click **Create Role**.
+
+## Step 2/4. Configure the Teleport IAM role mapping
+
+The next step is to give your Teleport users permissions to assume IAM roles in
+your Teleport cluster.
+
+You can do this by creating a Teleport role with the `aws_role_arns` field
+listing the IAM role ARN created in the previous step. Create a file called
+`aws-dynamodb-access.yaml` with the following content:
+
+```yaml
+kind: role
+version: v5
+metadata:
+  name: aws-dynamodb-access
+spec:
+  allow:
+    db_labels:
+      '*': '*'
+    db_users:
+    - '*'
```

Comment:
```suggestion
    - 'ExampleTeleportDynamoDBRole' # only the role name, do not specify the full AWS ARN
```

I realize that I suggested using the wildcard here at some point, but I changed my mind 😅 . I think we should just use the example role we've created in the guide, otherwise there's really no point in telling them to create a role instead of just using the default `access` role (which would already grant these permissions). It also gives us an opportunity to illustrate that they should not use the full role ARN, just the role's name.